static/src/components/setting/ownership_setting.vue

```diff
@@ -45,35 +45,103 @@
         <div class="col-md-8 pull-right">
           <div class="input-group">
             <input
-              v-model="search"
+              v-model="coownerQuery"
               type="text"
               class="form-control input-sm"
               placeholder="Try with full name or nick name"
             >
             <span class="input-group-append">
               <button
                 class="btn btn-sm btn-primary "
-                @click="searchUsers"
+                @click="searchCoowners"
               >
                 <i class="fa fa-search" />
                 Search
               </button>
             </span>
           </div>
           <div
-            v-if="searchResult.length"
+            v-if="coownerResult.length"
             class="dropdown-content"
           >
             <div
               class="scroll"
               style="max-height: 150px;"
             >
               <div
-                v-for="u in searchResult"
+                v-for="u in coownerResult"
                 :key="u.id"
                 :value="u"
                 class="row"
-                @click="add($event, u)"
+                @click="addCoowner($event, u)"
+              >
+                <p> {{ u['fullname'] }} </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="form-group row">
```

Comment:
The code of "co-owner" UI and "email contacts" UI look quite similar. Is it worth to extract it as a common VUE component which can accept different properties (co-owner or email contracts)?